holidayextras/ui-toolkit
# UXUI-369 Fix releasing of the docs

#### What does this PR do? (please provide any background)
Fixes the release-docs grunt task that we have. Currently if you run that you'll hit a problem where it can't open a file. I believe this to be because we are trying to copy so many files (all of node_modules).

As a result of that i've tidied up what actually gets pushed the gh-pages branch. We now only push everything in the `docs/public` folder instead of `docs/**/*`. 

I also updated the version of browserify in the `docs/package.json` to match the projects version.
#### What tests does this PR have?
n/a
#### How can this be tested?
Pull down this branch, clear out your existing node_modules folder. Run `npm install` and then `grunt release-docs`. It should publish just fine. Visit the [docs](http://hungrygeek.holidayextras.co.uk/ui-toolkit/) and check everything still works. Now run the docs locally(`grunt docs`) and check they still work.
#### Screenshots / Screencast
![](https://cldup.com/D8ca-lqTsy.png)
#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/gIz38XxQoAfO8/giphy.gif)
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
